testing/test_sampler.py

```diff
@@ -39,14 +40,14 @@ def _cmpt_label(ds_loader):
         },
     "ROICrop":
         {
-            "size": [48, 48],
+            "size": [128, 128],
             "preprocessing": True
         },
     "NumpyToTensor": {}
 }])
-@pytest.mark.parametrize('train_lst', [['sub-test001']])
+@pytest.mark.parametrize('train_lst', [['sub-unf01']])
 @pytest.mark.parametrize('target_lst', [["_lesion-manual"]])
-@pytest.mark.parametrize('roi_params', [{"suffix": "_seg-manual", "slice_filter_roi": 10}])
+@pytest.mark.parametrize('roi_params', [{"suffix": "_seg-manual", "slice_filter_roi": 16}])
```

Comment:
Why did you have to make that change?

Reply:
change was sue to misunderstanding removed in f6fed96